core/controllers/review_tests.py

```diff
@@ -17,16 +17,66 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+from constants import constants
 from core.controllers import acl_decorators
 from core.controllers import base
 from core.domain import skill_fetchers
 from core.domain import story_fetchers
 import feconf
 
 
+SCHEMA_FOR_CLASSROOM_URL_FRAGMENTS = {
+    'schema': {
+        'type': 'basestring',
+        'validators': [{
+            'id': 'is_regex_matched',
+            'regex_pattern': constants.VALID_URL_FRAGMENT_REGEX
+        }, {
+            'id': 'has_length_at_most',
+            'max_value': constants.MAX_CHARS_IN_CLASSROOM_URL_FRAGMENT
+        }]
+    }
+}
+
+SCHEMA_FOR_TPOIC_URL_FRAGMENTS = {
```

Comment:
Typo: topic.
Ditto everywhere this is used.

Reply:
Thank you, now moved them into constants since they are very commonly used even in other files